### PR TITLE
Font iteration 'bug'

### PR DESCRIPTION
**Pull request by msp**
See #126

### CODE REVIEW

File: fingerprint2.js [hunk 1]
```diff
@@ -359,13 +359,13 @@
         }
         var detect = function (font) {
             var detected = false;
-            for (var index in baseFonts) {
+            baseFonts.forEach(function() {
```
Comment (Valve) [inline fingerprint2.js]: AFAIK, this will not work in IE8

Comment (msp) [inline fingerprint2.js]: Looks like you're right:

http://caniuse.com/#search=forEach

I can change to a C style for loop..

Comment (Valve) [inline fingerprint2.js]: Thanks!

**Comment by JustinTArthur** [comment]
:+1: for this. For the sake of speed on older browsers, I'd recommend storing the length at loop initialization as is done here
https://github.com/Valve/fingerprintjs2/blob/62b276871fc67c9e58c8bdea03cc830ed8105c7c/fingerprint2.js#L422

**Comment by Valve** [comment]
Thanks, I will review and merge tomorrow morning!

**Comment by Valve** [comment]
Thanks, released 1.1.1

**Pull request merged**